src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -23,8 +23,8 @@ import cromwell.engine.{AbortRegistrationFunction, WorkflowDescriptor, _}
 import cromwell.logging.WorkflowLogger
 import cromwell.parser.BackendType
 import cromwell.util.StringUtil._
-import cromwell.util.TryUtil
-import cromwell.util.google.{GoogleCloudStorage, GoogleCloudStoragePath, GoogleCredentialFactory}
```

Comment:
I'm generally pro-conciseness (including here) but just curious what led to the change to Gcs